modules/ting/src/TingObject.php

```diff
@@ -17,38 +17,48 @@ class TingObject implements TingObjectInterface {
 
   // TODO BBS-SAL: We run with public access for now as some legacy-code still
   // needs directly access, but should be changed to protected when SAL is done.
-  public $id;
-  public $ownerId;
-  public $sourceId;
-  public $title;
-  public $shortTitle;
+  public $abstract;
```

Comment:
How about removing the comment above here?

We will not change the visibility of the properties due to all the automagic happening.

Reply:
Agreed